.github/workflows/docs.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.9
+          python-version: 3.x
```

Comment:
TIL [you can do this](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input)!